installer/manifests/keptn/charts/control-plane/values.yaml

```diff
@@ -203,7 +203,7 @@ configurationService:
     GIT_KEPTN_EMAIL: "keptn@keptn.sh"
   nodeSelector: {}
   gracePeriod: 60
-  preStopHookTime: 5
+  preStopHookTime: 20
```

Comment:
why configuration-service ?

Reply:
if for some reason it is still used instead of the resource-service it will very likely also need more time